pkg/kubelet/kubelet.go

```diff
@@ -2047,6 +2047,12 @@ func (kl *Kubelet) HandlePodReconcile(pods []*v1.Pod) {
 		// to the pod manager.
 		kl.podManager.UpdatePod(pod)
 
+		if status.NeedToReconcilePodReadiness(pod) {
+			start := kl.clock.Now()
```

Comment:
move `start := kl.clock.Now()` to the start of the `HandlePodReconcile` to be consistent with other functions.

Reply:
fixed